person.rb

```diff
@@ -20,3 +29,10 @@ def of_age?(age)
     age >= 18
   end
 end
+
+person = Person.new(22, 'maximilianus')
+puts person.correct_name
+capitalized_person = CapitalizeDecorator.new(person)
+puts capitalized_person.correct_name
+capitalized_trimmed_person = TrimmerDecorator.new(capitalizedPerson)
```

Comment:
Yeah, I see it now. I replaced the capitalizedPerson instance with its snake_case because of the linter but forgot to change it inside the method call. Thanks!